src/hooks/useProfiles.js

```
@@ -0,0 +1,170 @@
+
+const useProfiles = () => {
+  const [profiles, setProfiles] = useState([])
+  const [activeProfile, setActiveProfile] = useState(null)
+  const [isLoading, setIsLoading] = useState(false)
+
+  // Profilleri yükle
+  const loadProfiles = useCallback(async () => {
+    try {
+      setIsLoading(true)
+      const response = await fetch('/api/profiles')
+      const response = await fetch('/api/profiles')
```

Biome (2.1.2)

[error] Shouldn't redeclare 'response'. Consider to delete it or rename it.

'response' is defined here:


(lint/suspicious/noRedeclare)